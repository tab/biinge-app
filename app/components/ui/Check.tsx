import React from "react"
import { StyleProp, ViewStyle } from "react-native"
import Animated, { FadeIn } from "react-native-reanimated"

import colors from "styles/colors"
import Icon from "components/ui/Icon"

type Props = {
  style?: StyleProp<ViewStyle>
}

const CheckComponent = ({ style }: Props) => {
  return (
    <Animated.View entering={FadeIn} style={style}>
      <Icon name="checkmark-outline" color={colors.white} size={15} />
    </Animated.View>
  )
}

export default CheckComponent
