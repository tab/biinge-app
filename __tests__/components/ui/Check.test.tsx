import React from "react"
import "react-native"
import { render } from "@testing-library/react-native"

import { it } from "@jest/globals"

import Check from "components/ui/Check"

describe("Check", () => {
  it("renders correctly", () => {
    const tree = render(<Check />)
    expect(tree).toMatchSnapshot()
  })
})
