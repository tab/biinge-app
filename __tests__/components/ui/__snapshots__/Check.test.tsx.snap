// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check renders correctly 1`] = `
<View
  collapsable={false}
  entering={[Function]}
  jestAnimatedStyle={
    {
      "value": {},
    }
  }
  style={{}}
>
  <Text
    allowFontScaling={false}
    selectable={false}
    style={
      [
        {
          "color": "#ffffff",
          "fontSize": 15,
        },
        undefined,
        {
          "fontFamily": "Ionicons",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        {},
      ]
    }
  >
    î¬¨
  </Text>
</View>
`;
